docs/guide/ssr.md

```diff
@@ -1,5 +1,26 @@
 # SSR features
 
+## Initialize function
+
+An exported function inside `src/main.js`.
+
+This function contains the initial [context](/reference/) used to create the Vue instance.
+
+Example:
+
+```js
+export default function ({ app, router, store }) {
```

Comment:
`app` will be not defined here because we do `new Vue` on next line ;)

Reply:
nice catch :baseball: I looked at it too and thought about it :crying_cat_face:

Reply:
I could make note of the execution order and life-cycle availability of the context in the reference. It will be helpful for me. What do you think?

Reply:
Good idea ! I'll take it ! :+1: